include/internal/quic_wire.h

```diff
@@ -209,7 +209,7 @@ typedef struct ossl_quic_frame_new_conn_id_st {
     uint64_t              seq_num;
     uint64_t              retire_prior_to;
     QUIC_CONN_ID          conn_id;
-    unsigned char         stateless_reset_token[16];
+    unsigned char         stateless_reset_token[QUIC_STATELESS_RESET_TOKEN_LEN];
```

Comment:
Since you've now defined a `QUIC_STATELESS_RESET_TOKEN`, I'm OK if you want to change all existing constructs like this to use this wrapped type. But it's up to you.

Reply:
Part of the reason for wrapping the reset token in a struct was to allow for the possibility of also including a _valid_ or _set_ flag.  I'm not convinced we need that for MVP but might still later on.  I think the only case where this might be required is for a new connection that doesn't get a token set & this will default to all zero bytes.  I.e. pretty low risk of stumbling across it by mistake.

If we don't need it, I'd prefer to typedef the array directly or even avoid the typedef completely.

Reply:
That's fine.

FWIW, I'd rather avoid forcing all instances of a stateless reset token to have a 'valid' tag, because it doesn't really correspond with many wire formats and corresponding encoding functions where you are forced to always include a stateless reset token, without the ability to omit one. So this would create a state which is unrepresentable when passed to a wire format encoding function.

Reply:
I'm going to leave it wrapped in a struct.  Typedefed fixed sized arrays can be _surprising_ in C and the #define for the length is a really long identifier.

I'll update other uses to use the struct.